homepage/index.html

```diff
@@ -302,6 +316,17 @@ <h2>Crates</h2>
               <a href="https://github.com/uutils/rust-users">rust-users</a>
             </li>
           </ul>
+          <h2>Contributing</h2>
+          <p>You can help us out by:</p>
+          <ul>
+            <li>Contributing code</li>
+            <li>Contributing documentation</li>
+            <li>Reporting bugs (e.g. incompatibilities with GNU utilities)</li>
+            <li>Triaging bugs</li>
+            <li><a href="https://github.com/sponsors/uutils">Sponsoring uutils on GitHub</a></li>
+            <li>etc.</li>
```

Comment:
I would either remove "etc" or replace it with the additional points implied by "etc", though I think you already mentioned the main points.